### PR TITLE
Releases: add 0.16.2

### PR DESCRIPTION
**Pull request by harding**
Adds release notes and a short blog post.  Previews:

- Blog: http://dg0.dtrt.org/en/2018/07/27/release-0.16.2/
- Release notes: http://dg0.dtrt.org/en/releases/0.16.2/
- Download page: http://dg0.dtrt.org/en/download/

Todo:

- [x] Update files to correct dates for release:
    - `_releases/0.16.2.md` - change date in YAML header
    - `cd _posts/en/posts/ ; rename 's/.*release/'$(date -u -I)'-release/' _posts/en/posts/2018-07-27-release-0.16.2.md ; git add .`
- [x] Add magnet URI for torrent to _releases/0.16.2.md
- [x] If release notes change, update the release file with the changes

Note: this PR is expected to fail Travis CI as it expects the binaries to be in /bin/, which they're not yet.  The CI should pass once if Travis is restarted after the binaries are in their final location.

### CODE REVIEW

**Comment by harding** [comment]
Torrent downloads now, tests passed.  I'll add the magnet from #585.

This needs a ACK for its quick blog post before it should be merged.

**Comment by MarcoFalke** [comment]
@harding I can't find a blog post in the two files you edited.

**Comment by harding** [comment]
@MarcoFalke oh, wow, I feel dumb.  I never git added it.  It's in commit https://github.com/bitcoin-core/bitcoincore.org/pull/582/commits/06e0c67f3e5677a373277d3a674b1c80a66e65fd ; I'm going to squash this all and force push.

File: _posts/en/posts/2018-07-29-release-0.16.2.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- The [verifytxoutproof RPC][rpc verifytxoutproof] is no longer
+  vulnerable to a particular [expensive attack][tx-as-internal-node]
+  against SPV proofs publicly disclosed in early June.  The attack was
+  considered unlikely given that much cheaper attacks of roughly equal
+  effectiveness are well known.  Similarly, the [getblock RPC][rpc
+  getblock] also now returns extra information that can be used to
+  defeat this attack even if the requested block has been pruned.  None
+  of this mitigates the attack for actual SPV clients.
+
+- The [`abandontransaction`][rpc abandontransaction] RPC has been fixed
```
Comment (jnewbery) [inline _posts/en/posts/2018-07-29-release-0.16.2.md]: nit: inconsistent formating (links to the RPCs above cover the name of the RPC and the word RPC and do not have backticks to render as code)

File: _releases/0.16.2.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+### RPC and other APIs
+- #13451 `cbd2f70` rpc: expose CBlockIndex::nTx in getblock(header) (instagibbs)
+- #13507 `f7401c8` RPC: Fix parameter count check for importpubkey (kristapsk)
+- #13452 `6b9dc8c` rpc: have verifytxoutproof check the number of txns in proof structure (instagibbs)
+- #12837 `bf1f150` rpc: fix type mistmatch in `listreceivedbyaddress` (joemphilips)
+- #12743 `657dfc5` Fix csBestBlock/cvBlockChange waiting in rpc/mining (sipa)
+
+### GUI
+- #12432 `f78e7f6` [qt] send: Clear All also resets coin control options (Sjors)
+- #12617 `21dd512` gui: Show messages as text not html (laanwj)
```
Comment (jnewbery) [inline _releases/0.16.2.md]: Link to commit fails. Linking to `21dd5127` works though.

**Pull request merged**